subprojects/packagefiles/backward-cpp/meson.build

```diff
@@ -0,0 +1,16 @@
+project('backward-cpp', 'cpp', version: '1.6')
+
+install_headers('backward.hpp', subdir : 'backward')
+backward = library('backward-cpp', files('backward.cpp'), include_directories: '.', install: true)
```

Comment:
You don't need `include_directories: '.'`. The current source and build directories are put in the include path by default.